config/metricman.schema.exs

```diff
@@ -4,15 +4,15 @@
       doc: """
       Influxdb configuration in form of <shema>://<host>[:<port>]/<database>
       """,
-      to: "exometer_core.report.reporters",
+      to: "exometer_core.report.reporters.db",
```

Comment:
I think this is a little bit dangerous because you break into the namespace of the reporters. Usually "exometer_core.report.reporters" contains the options for *all* reporters and now suddenly you place some options here which are specific for a single reporter :).